WalletLibrary/WalletLibrary/Networking/Entities/OpenID4VCI/PreAuthTokenResponse.swift

```diff
@@ -15,5 +15,5 @@ struct PreAuthTokenResponse: Decodable
     let token_type: String?
     
     /// When the token expires.
```

Comment:
Can you specify the time unit used here? If its `exp` from JWT https://www.rfc-editor.org/rfc/rfc7519#section-2 its seconds since epoch.